src/Compilers/CSharp/Test/Semantic/Semantics/PatternSwitchTests.cs

```diff
@@ -3265,5 +3277,113 @@ void M()
             Assert.Equal(SpecialType.System_Boolean, type.Type.SpecialType);
             Assert.Equal(SpecialType.System_Boolean, type.ConvertedType.SpecialType);
         }
+
+        [Fact, WorkItem(20210, "https://github.com/dotnet/roslyn/issues/20210")]
+        public void SwitchOnNull_20210()
+        {
+            var source =
+@"class Sample
+{
+    void M()
+    {
+        switch (default(object))
+        {
+          case bool _:
```

Comment:
Nit: the formatting in this code snippet is off (using 2 spaces in some places?)

Reply:
That is a style supported by VS (I think), and used by the customer who submitted this test.